src/internal/connector/exchange/event_calendar_cache.go

```diff
@@ -73,6 +73,10 @@ func (ecc *eventCalendarCache) Populate(
 		return errors.Wrap(err, "establishing calendar paths")
 	}
 
+	if err := ecc.populatePaths(ctx); err != nil {
+		return errors.Wrap(err, "establishing calendar paths")
+	}
+
```

Comment:
```suggestion
```

Reply:
I might be blind, but I don't see any changes in your suggestion...

Reply:
these lines are a repeat of the ones above so they should be removed not added 😄 I agree it's a little confusing just looking at the diff. I guess I should have left a note too

Reply:
ooh, gotcha.  I guess if I had looked at the changes instead of just the comment, I would have figured it out.